sdk/v2/data.go

```diff
@@ -38,8 +39,8 @@ func NewEntityData(entityName, entityType string) (EntityData, error) {
 	d := EntityData{
 		// empty array or object preferred instead of null on marshaling.
 		Metrics:   []metric.MetricSet{},
-		Inventory: Inventory{},
-		Events:    []*Event{},
+		Inventory: v1.Inventory{},
```

Comment:
This is confusing for me. I understand that v1 refers to protocol 1, but this is also the part of Protocol 2 and it will be always like this. This will be confusing for the users. They won't know what's coming from protocol 1, and they won't understand why they need to use sth from v1 in v2. 
I wouldn't to this change (I mean the whole PR doesn't make sense for me). But this is just my opinion.
/cc @smoya, @areina

Reply:
I don't see it confusing, since protocol v2 uses protocol v1 as base. But If the rest of the team prefers discarding this commit, I'm ok with it.

Reply:
Also users can migrate easier to protocol v2, since they can reuse their existing types from Protocol v1.

Reply:
Anyway, in the last commit, I tried to get an intermediate solution:

```go
package v2

// Inventory is the data type for inventory data produced by an integration data
// source and emitted to the agent's inventory data store.
type Inventory v1.Inventory

// Event is the data type to represent arbitrary, one-off messages for key
// activities on a system.
type Event v1.Event
```